Source/VSProps/QtCompile.props

```diff
@@ -32,6 +32,12 @@
       This isn't too bad since we live with the fact that Qt-using projects will have different compiler args already.
       -->
       <AdditionalOptions>%(AdditionalOptions) /wd4946</AdditionalOptions>
```

Comment:
Is this line still needed if all warnings are suppressed?  (Same in CMakeLists.txt)

Reply:
Is that for the qt headers or for the project itself?  I assumed it was for the project itself.

Reply:
Looks like it is for the headers.  Going to try it and confirm